src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -7,6 +7,18 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
 .FileUpload {
   padding: $fileupload-padding;
   text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
```

Comment:
The new illustration makes the content smaller then the `DropZone` `min-height`. This vertically centers the content and expands it to the parent container height.